### ISSUE TITLE
Test keyboard operation

### ISSUE THREAD

**Issue by sailakarjalainen** [otavanopisto/muikku#5084]
All functionality should be operable through a keyboard interface without requiring specific timings for individual keystrokes *except where the underlying function requires input that depends on the path of the user's movement and not just the endpoints. 

Also:
If keyboard focus can be moved to a component of the page using a keyboard interface, then focus can be moved away from that component using only a keyboard interface. If it requires more that unmodified arrow or tab keys or other standard exit methods, the user must be advised of the method. All content must meet this success criterion, otherwise it interferes with with a user’s ability to use the whole page.

--> Test all views. Student views may have higher priority. 

Possible problems (?): 
- communicator: sending and answering messages
- exercises: multiple choice options, choosing an option, organizer field, sorter field
- picture slider in open front page
- marticulation examination?
etc.

Some iframes in materials are known fails, no need to test.

Problems that can't be fixed right now must be listed in accessibility statement.

Success Criteria: 
2.1.1, level A ([FI](http://papunet.net/saavutettavuus/wcag-ohjeet/211-nappaimisto), [EN](https://www.w3.org/TR/WCAG21/#keyboard)) (*no exception in criterion 2.1.3 (AAA)
2.1.2, level A ([FI](http://papunet.net/saavutettavuus/wcag-ohjeet/212-ei-nappaimistoa-ansa), [EN](https://www.w3.org/TR/WCAG21/#no-keyboard-trap))

**Comment by sailakarjalainen** [comment]
Näppäimistökäyttöön liittyy AA-tason vaatimus siitä, että käyttäjän tulee nähdä, missä elementissä fokus on. Kannattaako tämä testata samalla? https://papunet.net/saavutettavuus/wcag-ohjeet/247-nakyva-kohdistus
